akka-projection-r2dbc/src/main/resources/reference.conf

```
@@ -21,6 +21,12 @@ akka.projection.r2dbc {
     # within this time window from latest offset.
     time-window = 5 minutes
 
+    # Backtracking window of the source (query). Should be equal to
+    # the akka.projection.r2dbc.offset-store.time-window that is used for the
```

Reference to the wrong setting, should be equal to the query backtracking window?

```suggestion
    # the akka.persistence.r2dbc.query.backtracking.window that is used for the
```